integration/network/inspect_test.go

```diff
@@ -2,21 +2,115 @@ package network // import "github.com/docker/docker/integration/network"
 
 import (
 	"context"
+	"net"
 	"testing"
 	"time"
 
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/api/types/filters"
 	swarmtypes "github.com/docker/docker/api/types/swarm"
 	"github.com/docker/docker/client"
+	"github.com/docker/docker/integration/internal/container"
 	"github.com/docker/docker/integration/internal/network"
 	"github.com/docker/docker/integration/internal/swarm"
+	"github.com/docker/docker/internal/test/request"
 	"gotest.tools/assert"
+	is "gotest.tools/assert/cmp"
 	"gotest.tools/poll"
+	"gotest.tools/skip"
 )
 
 const defaultSwarmPort = 2477
 
+func TestNetworkFilter(t *testing.T) {
```

Comment:
This is a weird test :sweat_smile: Shouldn't it be : get the id from `NetworkInspect`, filter the network list by the name `bridge` _and_ assert it's the same id ?

Reply:
Thank you for the review. Fixed.